Jakefile.js

```diff
@@ -658,19 +658,21 @@ function exec(cmd, completeHandler, errorHandler) {
 }
 
 function cleanTestDirs() {
-    // Clean the local baselines directory
-    if (fs.existsSync(localBaseline)) {
-        jake.rmRf(localBaseline);
-    }
+    if (!environmentVariableIsDisabled("CLEAN_TESTS")) {

```

Comment:
instead of checking an environment variable to skip cleaning, a separate task called "runtest-dirty" or "runtest-noclean" would be safer. At least if somebody does $env:CLEAN_TESTS = "no" and forgets, they'll get very confusing baseline-accept runs later.